fishtank.py

```diff
@@ -109,6 +116,33 @@ def move(self, walls, goblins):
             elif self.change_y < 0 and self.rect.top != goblin.rect.top:
                 self.rect.top = goblin.rect.bottom
 
+    def pick_target(self, possible_targets):
```

Comment:
Moved pick_target to Organism. In theory Ogre can use it to find Goblins, but it doesn't do that yet.